net-libs/miniupnpc/miniupnpc-2.0.20180222.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="UPnP IGD client lightweight library"
+DESCRIPTION="The usage of the miniUPnP client library is useful whenever an \
+application needs to listen for incoming connections.
+Examples : P2P applications, FTP clients for active mode, IRC (for DCC)
+or IM applications, network games, any server."
+HOMEPAGE="http://miniupnp.free.fr"
+COPYRIGHT="2005-2018 Thomas Bernard"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://miniupnp.free.fr/files/miniupnpc-$portVersion.tar.gz"
+CHECKSUM_SHA256='587944686469d09f739744b3aed70c7ce753a79c40d6f1227f68a3e962665b75'
+PATCHES="miniupnpc-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	miniupnpc$secondaryArchSuffix = $portVersion
+	lib:libminiupnpc$secondaryArchSuffix = 2.0.0 compat >= 2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	miniupnpc${secondaryArchSuffix}_devel = $portVersion
+	devel:libminiupnpc$secondaryArchSuffix = 2.0.0 compat >= 2
+	"
+REQUIRES_devel="
+	miniupnpc$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:cmake
+	cmd:make
+	"
+
+defineDebugInfoPackage xapian$secondaryArchSuffix \
```

Comment:
```diff
-defineDebugInfoPackage xapian$secondaryArchSuffix \
+defineDebugInfoPackage miniupnpc$secondaryArchSuffix \
```

BTW, maybe it would be a good idea to define a `libVersion` and `libVersionCompat`.